src/lib.rs

```diff
@@ -519,6 +541,7 @@ impl Build {
         }
```

Comment:
Technically if you are no longer inheriting CFLAGS/CXXFLAGS then this is no longer needed, but this is arcane enough knowledge and sufficiently relevant that it probably deserves to be preserved as a comment.